### PR TITLE
tea -> pkgx for ci/cd

### PR DESCRIPTION
**Pull request by getneil**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -118,7 +112,7 @@ jobs:
       - name: get gui version
         id: gui-version
         run: |
-          tea +stedolan.github.io/jq
+          pkgx +stedolan.github.io/jq
           export version=$(echo $(cat package.json) | jq --raw-output .version)
```
Comment (mxcl) [inline .github/workflows/ci.yml:116]: could avoid line 115 and just do `pkgx jq --raw…` here

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -97,18 +97,18 @@ jobs:
       - name: cp latest as previous release
         run: |
           aws s3 cp \
-            s3://dist.tea.xyz/tea.xyz/gui/tea-latest-arm64.dmg \
-            s3://dist.tea.xyz/tea.xyz/gui/tea-previous-arm64.dmg
+            s3://preview.gui.tea.xyz/release/ossapp-latest-arm64.dmg \
+            s3://preview.gui.tea.xyz/release/ossapp-previous-arm64.dmg
```
Comment (mxcl) [inline .github/workflows/release.yml:101]: put the bucket id in a secret

**Review by mxcl — CHANGES_REQUESTED** [review]
almost

File: .github/workflows/release.yml (hunk 2 above)
Comment (getneil) [reply .github/workflows/release.yml]: applied all of this.

**Review by mxcl — APPROVED** [review]

**Pull request merged**